modules/vaos/spec/request/appointment_requests_request_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe 'vaos appointment requests', type: :request do
+  include SchemaMatchers
+
+  let(:rsa_private) { OpenSSL::PKey::RSA.generate 4096 }
+
+  before do
+    Flipper.enable('va_online_scheduling')
+    sign_in_as(current_user)
+    allow_any_instance_of(VAOS::JWT).to receive(:rsa_private).and_return(rsa_private)
```

Comment:
Once the user service PR goes in this could change to 
```ruby
allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
```